packages/dd-trace/test/appsec/iast/vulnerability-formatter/index.spec.js

```diff
@@ -16,6 +22,14 @@ function doTest (testCase, parameters) {
     })
   }
 
+  if (expected.vulnerabilities?.length && excludedVulnerabilityTypes.includes(expected.vulnerabilities[0].type)) {
+    return
+  }
+
+  if (excludedTests.includes(description)) {
+    return
+  }
+
```

Comment:
should these ifs be in a filtering function? something like 

```javascript
function filterByTypeAndDescription (testCase){
  if (testCase.type !== 'VULNERABILITIES' ||
    excludedVulnerabilityTypes.includes(testCase.input[0]?.type) ||
    excludedTests.include(testCase.description)) {
    return false
  } else {
    return true
  }
}

suite.filter(filterByTypeAndDescription)
```

Reply:
not easily. Keep in mind that the final description is filled few lines above. They are not filled when we are filtering suite in line 80.